media-server/config.txt

```diff
@@ -1,10 +1,11 @@
 CAMERA_DEVICE='/dev/video0'
 CAMERA_DEVICE_WIDTH=1280
 CAMERA_DEVICE_HEIGHT=720
-MODEL_LOCATION=/home/coral/mobilenet_v2_1.0_224_quant_edgetpu.tflite # Use absolute path.
```

Comment:
What if we create 2 config files? one call `config_classifier.txt` and other `config_detector.txt` ? So the user don't have to mess modifying the config files and can test both easily